src/components/visualizations/headline/utils/HeadlineTransformationUtils.ts

```diff
@@ -26,7 +27,7 @@ export interface IHeadlineExecutionData {
 export interface IHeadlineDrillItemContext {
     localIdentifier: AFM.Identifier;
     value: string;
-    element: VisElementType;
+    element: HeadlineElementType;
```

Comment:
`elementType` ?

Reply:
I woud rather to not change it since it is used on many places and is basic structure of drill events - we would need some adapter to maintain compatibility.